huaweicloud/services/cbr/resource_huaweicloud_cbr_vault.go

```diff
@@ -228,62 +216,72 @@ func ResourceVault() *schema.Resource {
 }
 
 func buildAssociateResourcesForServer(rType string, resources []interface{}) ([]vaults.ResourceCreate, error) {
+	results := make([]vaults.ResourceCreate, len(resources))
 	if len(resources) == 0 {
-		return []vaults.ResourceCreate{}, nil
+		// The argument `resources` is required.
+		return results, nil
```

Comment:
always return results， you can delete this line

Reply:
OFC